docs/conf.py

```
@@ -0,0 +1,195 @@
+copybutton_prompt_is_regexp = True
+
+# -- Options for HTML output -------------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = "pydata_sphinx_theme"
+github_repo = "rtems-proxy"
+github_user = "epics-containers"
+switcher_json = f"https://{github_user}.github.io/{github_repo}/switcher.json"
+switcher_exists = requests.get(switcher_json).ok
```

Ruff (0.16.0)

[error] Probable use of `requests` call without timeout

(S113)